Adding dashboard hardhat plugin docs (#1408)
trufflesuite/trufflesuite.com: src/docs/truffle/how-to/use-the-truffle-dashboard.md

<p class="alert alert-info">
<i class="far fa-info-circle"></i> <strong>Information</strong>: Truffle Dashboard cannot decode requests

> when using tools other than Truffle or Hardhat (with the plugin outlined below); we plan to add continue adding support for other tools in the future.

Reviewer: ```suggestion
when using tools other than Truffle or Hardhat (with the plugin outlined below); we plan on continue adding support for other tools in the future.
```

Reviewer 2: still reads odd. drop `continue` in @cliffoo 's suggestion so it's `...plan on adding support ...`

> when using tools other than Truffle or Hardhat (with the plugin outlined below); we plan to add continue adding support for other tools in the future.

Reviewer 3: Not to pile on, but here's my suggestion. :) 

```suggestion
when using tools other than Truffle or Hardhat (with the plugin outlined below). We plan to add support for additional tools in the future.
```

Author: Awesome, added 👍

</p>

### Using Truffle Dashboard with the Hardhat plugin

> The plugin brings the complete Truffle Dashboard experience, including decoded transaction information, to your Hardhat projects. See the [Installation](https://github.com/trufflesuite/truffle/tree/develop/packages/dashboard-hardhat-plugin#installation) steps to get it setup for your project. It extends `npx hardhat compile` by sending the compiled artifacts to Truffle Dashboard, enabling seemless integrate with your existing workflow.

Reviewer: ```suggestion
The plugin brings the complete Truffle Dashboard experience, including decoded transaction information, to your Hardhat projects. See the [Installation](https://github.com/trufflesuite/truffle/tree/develop/packages/dashboard-hardhat-plugin#installation) steps to get it setup for your project. It extends `hardhat compile` by sending the compiled artifacts to Truffle Dashboard, enabling seamless integration with your existing workflow.
```

> The plugin brings the complete Truffle Dashboard experience, including decoded transaction information, to your Hardhat projects. See the [Installation](https://github.com/trufflesuite/truffle/tree/develop/packages/dashboard-hardhat-plugin#installation) steps to get it setup for your project. It extends `npx hardhat compile` by sending the compiled artifacts to Truffle Dashboard, enabling seemless integrate with your existing workflow.

Reviewer 3: ```suggestion
The plugin brings the complete Truffle Dashboard experience, including decoded transaction information, to your Hardhat projects. See the [Installation](https://github.com/trufflesuite/truffle/tree/develop/packages/dashboard-hardhat-plugin#installation) steps to get it setup for your project. The Hardhat plugin extends `npx hardhat compile` by sending the compiled artifacts to Truffle Dashboard, enabling seamless integrate with your existing workflow.
```

Author: Awesome thanks @fainashalts
